web-plugin/js/common.js

```diff
@@ -0,0 +1,24 @@
+class TofuItem {
+    constructor(name, unit, quantity) {
+        this.name = name;
+        this.quantity = quantity;
+        this.unit = unit;
+    }
+    toString() {
+        return `TofuItem Name: ${this.name} quantity: ${this.quantity} unit: ${this.unit}`;
+    }
+}
+
+const STATUS = {
+    Empty: 'empty',
+    HaveFood: 'offset',
+    ERROR: 'error'
+}
+
+//we need a convert table for like  "ct -> ounce". many convert GHGI support it.
```

Comment:
```suggestion
//TODO: we need a convert table for like  "ct -> ounce". many convert GHGI support it.
```

Reply:
yeah, below is the table. :)  that comments can changed to a description.